cmd/operator/main.go

```diff
@@ -114,6 +127,11 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	log := logrus.New()
```

Comment:
Why do we use `setupLog` at all here?
I'd prefer we either change the `setupLog` uses below this line to `log` lines or just leave this where it was and use `setupLog` for this like the surrounding calls.